bucket/authy.json

```diff
@@ -0,0 +1,63 @@
+{
+    "version": "1.7.2",
+    "description": "Free desktop app for two factor authentication.",
+    "homepage": "https://authy.com/",
+    "license": "Freeware",
+    "architecture": {
+        "64bit": {
+            "url": "https://s3.amazonaws.com/authy-electron-repository-production/authy/stable/1.7.2/win32/x64/authy-electron-1.7.2-full.nupkg",
+            "hash": "sha1:53646438bd9c05efa4790b655ab1d056302a6a7f"
+        },
+        "32bit": {
+            "url": "https://s3.amazonaws.com/authy-electron-repository-production/authy/stable/1.7.2/win32/x32/authy-electron-1.7.2-full.nupkg",
+            "hash": "sha1:adb40cd6a325585b3b2fc32c4ccf29d8d1bb08da"
+        }
+    },
+    "extract_dir": "lib\\net45",
+    "pre_install": [
+        "$appdata = \"$env:APPDATA\\Authy Desktop\"",
+        "if (!(Test-Path $persist_dir)) {",
+        "   New-Item -ItemType Directory -Path $persist_dir | Out-Null",
+        "}",
+        "if (Test-Path $appdata) {",
+        "   Remove-Item -Force -Recurse $appdata | Out-Null",
+        "}",
+        "New-Item -ItemType Junction -Path $appdata -Target $persist_dir | Out-Null"
+    ],
+    "bin": [
+        [
+            "Authy Desktop.exe",
+            "authy"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "Authy Desktop.exe",
+            "Authy Desktop"
+        ]
+    ],
+    "uninstaller": {
```

Comment:
Why would you do this? So much not needed operations on each update. Removing and then linking...